src/Path.Drag.js

```diff
@@ -18,7 +18,7 @@ L.PathDraggable = L.Draggable.extend({
 
   _onDown: function (e) {
     var first = e.touches ? e.touches[0] : e;
-    this._startPoint = new L.Point(first.clientX, first.clientY);
```

Comment:
Don't really get the finality of those comments.
My point is that `this._startPoint` will then be compared with other points (cf https://github.com/Leaflet/Path.Drag.js/pull/4/files#diff-50ed3b9a9a28a38d8a12041bc4d297daR69), and thus those other points needs to be translated to the same x/y.